src/main/java/com/ethercamp/harmony/web/controller/ContractsController.java

```diff
@@ -40,87 +44,106 @@
  */
 @Slf4j
 @RestController
+@RequestMapping("/contracts")
+@Conditional(WebEnabledCondition.class)
```

Comment:
Would it be possible to mark spring config class with `@Conditional` annotation? The idea is to have a default config and one config per module that defines all the things related to that module.
This would decrease number of module definition settings across the codebase